bundle/Resources/config/ibexa_seo_toolkit_images.yaml

```diff
@@ -0,0 +1,12 @@
+ezpublish:
+    system:
+        default:
+            image_variations:
+                ibexa_seo_toolkit_og_image1200x630:
+                    filters:
+                        thumbnail:
+                            size: [ 1200, 630 ]
+                            mode: outbound
+                    post_processors:
```

Comment:
You shouldn't add the postprocessors in the bundle's config as they aren't required to install the bundle or Ibexa